### ISSUE TITLE
[Alert] Reduce right padding on va-alert component

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/vets-design-system-documentation#658]
# Component improvement request

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## This update is for:

- [ ] Content style guide
- [x] Component
- [ ] Pattern
- [ ] Utility
- [ ] Other

## What is the name?

`va-alert`

## What is the nature of this update?
I propose reducing the padding in the `div.alert` CSS declaration from `padding: 3.2rem 6.4rem 3.2rem 2.4rem` to `padding: 2rem`, which aligns with the background only component.

If that is for some reason going too far, at a minimum, we should update the declaration to `padding: 3.2rem 2.4rem` to reduce the right side padding

## Additional Context

`6.4 rem` for right padding feels excessive, and I can't think of a good reason to have that much padding on the left side. I think maybe it's there to balance out the gutter under the icon? 

This especially impacts mobile devices. It causes alerts to be taller, and requires more scrolling - here's an example from an EDU application form from Chrome emulator. The alert doesn't fit in the viewport.  These short lines of text would be especially taxing for folks who are zoomed in or use magnification.

![Screen Shot 2022-03-02 at 2 59 01 PM](https://user-images.githubusercontent.com/55992116/156441568-04fa7f9f-8967-48f1-be5b-951f5bf2154d.png)

It also causes alerts with 47 characters or more to wrap onto two lines. That causes them to take up more vertical space than should be needed for a brief message.

Here's the same alert above with the declaration updated to 2rem. Now it fits in the viewport, and the padding doesn't look terrible IMO. I can share more examples if needed.

![Screen Shot 2022-03-02 at 2 59 20 PM](https://user-images.githubusercontent.com/55992116/156443249-ffcfdbbe-0f5a-4654-af07-00ee32ac8236.png)

**Comment by andaleliz** [comment]
@caw310 can you please confirm that this ticket is on your radar? If you can provide any kind of next steps that'd be great.

**Comment by caw310** [comment]
@andaleliz it's in our backlog to be prioritized.

**Comment by KevinMHoffmanUSDS** [comment]
@andaleliz Good morning! I was reviewing this and I wanted to make sure I understood your proposal correctly. 

The title says 

> Reduce right padding 

but the ticket states

> 6.4 rem for left padding feels excessive, and I can't think of a good reason to have that much padding on the left side.

Which are you proposing changing, and what is the problem that this solves for a user? Not aesthetically, but specifically "a VA.gov user will be able to more easily..." 

Thanks!

**Comment by caw310** [comment]
@andaleliz I've added 'what problem does this solve to the issue template.

**Comment by andaleliz** [comment]
@caw310 thank you! I think that'll be very useful for issue authors and readers going forward.

**Comment by k80bowman** [comment]
For the engineer who picks this up, let's tackle #810 along with this ticket.
